sink/sink

```diff
@@ -210,12 +211,21 @@ class Badger(object):
             with open(os.path.join(self.location, name + ".html"), 'wb') as f:
                 f.write('<html><head><meta http-equiv="refresh" content="0; {url}"></head></html>'.format(url=data['link']))
 
+class Extras(object):
+    def __init__(self, config):
+        pass
+
+    def push(self, data):
+        if 'extras' in data:
+            for url in data['extras']:
+                urllib.urlretrieve(url, os.path.basename(url))
```

Comment:
If this fails, the logging shouldn't fail right? Should we catch exceptions here?